spec/requests/entrypoint_spec.rb

```diff
@@ -47,6 +47,7 @@
         "version"     => Vmdb::Appliance.VERSION,
         "build"       => Vmdb::Appliance.BUILD,
         "appliance"   => MiqServer.my_server.name,
+        "time"        => a_kind_of(String),
```

Comment:
Do you know what format this time will be in? Perhaps you could leverage timecop around this test to make an assertion on it? I would expect that it would come back in ISO8601 format 😁

Reply:
come to think of it, it should be consistent with other time attrs the API normalizes, see normalize_time(), so Time.now.utc.iso8601 to be specifically.